README.md

```diff
@@ -581,8 +591,12 @@ Other Style Guides
       'of Batman. When you stop to think about how Batman had anything to do ' +
       'with this, you would get nowhere fast.';
 
-    // good
+    // not too good
```

Comment:
also please revert this. template literals are not permitted without interpolations.